README.md

```diff
@@ -32,7 +32,7 @@ pip install patch-api
 
 ### Requirements
 
-- Python 3.6.1
+- Python 3.10
```

Comment:
As discussed on https://github.com/patch-technology/client-code-generation/pull/70#pullrequestreview-1359412217, are we sure we want to deprecate Python < 3.10. Seems a bit aggressive? (but I'm not up-to-date on the Python ecosystem)